src/kernel/src/vm/mod.rs

```diff
@@ -622,7 +622,7 @@ impl Vm {
             return Err(SysErr::Raw(EINVAL));
         }
 
-        let slice_size = num_of_ops.try_into().ok().ok_or(SysErr::Raw(EINVAL))?;
+        let slice_size = num_of_ops.try_into().map_err(|_| SysErr::Raw(EINVAL))?;
```

Comment:
I can't find this check on the PS4.

Reply:
![image](https://github.com/obhq/obliteration/assets/48620541/9e2af809-6366-4e26-a73c-8e0f39a3b7ec)

I refactored it